### DISCUSSION TITLE
Genre showing Numeric ID3v1 Tag instead of String

### DISCUSSION THREAD

**Discussion by MickMorley** [jcorporation/myMPD#996] [Q&A] [answered]
When browsing through my library under Genre, there is 1 song and 1 album showing ID3v1 numeric equivalent instead of the actual genre string.

I did confirm that 2 is country and 79 is hard rock and that they are correct in the mp3 file.  Is this a MyMPD issue or MPD issue?

![image](https://user-images.githubusercontent.com/19698105/224418900-44d89f78-9d0d-48b7-b377-1d4d0a17f627.png)

![image](https://user-images.githubusercontent.com/19698105/224420374-8cf6480d-e966-428f-bdf6-71cac12e38f9.png)

**Comment by jcorporation** [comment]
myMPD handles the genres listed by MPD. Query MPD directly to check if it is a MPD issue, e.g. with mpc.

The most likely cause is a invalid tagged file.

**Reply by MickMorley** [reply]
Thanks I wasn't sure which program had the issue.

It is a `MPD` issue and I found that is was `ID3v2.4` tags.  I re-saved the files using Mp3tag (v3.19) and it saved the tag as a `ID3v2.3` and `MPD` now sees the genre as `Country` instead of `(2)`

![image](https://user-images.githubusercontent.com/19698105/224460696-574e3754-9898-4bec-a46c-b9a8daf678de.png)

**Comment by MickMorley** [comment, marked as answer]
Solution: `MPD` does not render Genre tags correctly from `ID3v2.4`.  Change them or re-save them in a program like `Mp3tag` which will save to an `ID3v2.3` format.

**Comment by remus-selea** [comment]


**Reply by jcorporation** [reply]
This question was already answered and your comment has nothing todo with the initial question.

**Reply by remus-selea** [reply]
Got it, no worries. I deleted it for being off-topic.
